### PR TITLE
setup tokio-console

### PR DESCRIPTION
**Pull request by ermalkaleci**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ Quick start: `cargo run -- --config config.yml`
 
 This will run a proxy server with [config.yml](config.yml) as the configuration file.
 
+Run with `RUSTFLAGS="--cfg tokio_unstable"` to enable [toko-console](https://github.com/tokio-rs/console)
```
Comment (chevdor) [inline README.md:13]:
```suggestion
Run with `RUSTFLAGS="--cfg tokio_unstable"` to enable [tokio-console](https://github.com/tokio-rs/console)
```

**Review by chevdor — CHANGES_REQUESTED** [review]
typo :)

File: README.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ Quick start: `cargo run -- --config config.yml`
 
 This will run a proxy server with [config.yml](config.yml) as the configuration file.
 
+Run with `RUSTFLAGS="--cfg tokio_unstable"` to enable [tokio-console](https://github.com/tokio-rs/console)
```
Comment (xlc) [inline README.md:13]: why it is not a feature?

Comment (ermalkaleci) [reply README.md:13]: Because tokio_unstable is needed to enable experimental api so I thought with one flag we can turn on tokio console as well

**Pull request merged**